docker-runner.mjs

```diff
@@ -0,0 +1,31 @@
+import {spawn} from 'child_process';
+
+let grist;
+
+function startGrist(newEnv) {
+  const env = newEnv ? {...process.env, ...newEnv} : process.env;
```

Comment:
Could you save yourself a ternary here by setting `newEnv = {}` in the parameters, and just having `env = { ...process.env, ...newEnv}`?

Reply:
Oh, right, thanks, that's cleaner.